JamfPro/JamfPro/JamfPro.pq

```diff
@@ -1,260 +1,110 @@
-            { "Computers - Configuration Profiles", computerConfigurationProfilesImpl(jamfUrl, temp_table), "Table", "Table", true},
-            { "Mobile Devices", MobileDevicesImpl(temp_mobile_devices[mobileDevices]), "Table", "Table", true },
-            { "Computer Device Groups", ComputerGroupsImpl(temp_comp_group[computerGroups]), "Table", "Table", true },
-            { "Mobile Device Groups", MobileDevicesGroupsImpl(temp_mobile_devices_group[deviceGroups]), "Table", "Table", true },
-            {"Mobile Devices - ExtensionAttributes",MdExtensionAttributes(temp_mobile_devices[mobileDevices]),"Table","Table",true},
-            {"Computers - ExtensionAttributes",extensionAttributes(url, temp_table),"Table","Table",true},
-            {"Mobile - Applications",MobileApplicationsImpl(temp_mobile_devices[mobileDevices]),"Table","Table",true},
-            {"Computers - Applications", computerApplicationsImpl(url, temp_table), "Table", "Table", true}
-        }),
-        navTable = Table.ToNavigationTable(source, {"Name"}, "Name", "Data", "ItemKind", "ItemName", "IsLeaf"),
-        test_table = computersImpl(jamfUrl, temp_table)
-        
-        
-    in
-        test_table;
-
-/*
-This validates if the provided URL is in the correct format.
-*/
-ValidateUrlScheme = (url as text) as text => if (Uri.Parts(url)[Scheme] <> "https") then error "Url scheme must be HTTPS" else url;
-
-
-/*
-This initializes the table to be used globally for the LegacyAPINavTable.
-*/
-legacyInitializeGlobalRecord = (url as text) as any =>
-let
-        response = JSSResource(url, "/computers"),
-       jsonComputers=Json.Document(response),
-       computers=Table.FromRecords(jsonComputers[computers]),
-       computersModified=Table.AddColumn(computers,"computerDetails", each GetComputerDetails([id],url)),
-       computerApps=Table.AddColumn(computers,"computerDetails",each GetComputerApps([id],url)),
-      return_record=Record.Combine({[computers=computersModified,computerApps=computerApps]})
-in
-        return_record;
+// simple function to validate that URL is in correct format
+validateUrlScheme = (url as text) as text => if (Uri.Parts(url)[Scheme] <> "https") then error "Url scheme must be HTTPS" else url;
 
-/*
-Creates a table of pages to be used as reference for future queries to the /v1/computers-inventory endpoint
-
-note: Uses 100 devices per page as default
-*/
-initializeGlobalRecord = (url as text) as any =>
+// initialization functions: 
+// these functions set the page tables for the global records
+initializeComputerRecord = (url as text) as table =>
     let
-        devices = UAPIResource(url, "uapi/v1/computers-inventory?page_size=1")[totalCount],
-        // We divide by 100 since the default amount per page is 100
+        // number of total devices in computer inventory record
+        devices_count = UAPIResource(url, "uapi/v1/computers-inventory")[totalCount],

```

Comment:
We can change these URLs to be `api/...` instead of `uapi/...`. The "UAPI" thing is legacy--still supported, but not what we document anymore.